src/api/endpoint/api/package.ts

```diff
@@ -88,15 +108,15 @@ export default function (
     function (req: $RequestExtend, res: $ResponseExtend): void {
       const { scopedPackage, filename } = req.params;
 
-      downloadStream(scopedPackage, filename, storage, req, res);
+      downloadStreamOrRedirect(scopedPackage, filename, storage, req, res, config);
```

Comment:
let's do the following adding this an experiment, but we need some changes on this method. Let's keep the experiment entry point isolated as possible.

```
  if (_.get(config, 'experiments.tarball_url_redirect') === true) {
    redirectMethodLogic();
  } else {
    originalMethod();
 }
```
We won't need this anymore https://github.com/verdaccio/monorepo/pull/320/files since experiments are not part of types.

Reply:
Something else add to https://github.com/favoyang/verdaccio/blob/tarball-url-redirect/conf/default.yaml#L79 the experiment with some notes to this PR disable as the others, for `default.yaml and `docker.yaml`

Reply:
Fixed